pom.xml

```diff
@@ -2078,6 +2078,13 @@
         </property>
       </activation>
     </profile>
+
+    <profile>
+      <id>hudi-metaserver</id>
+      <modules>
+        <module>hudi-metaserver</module>
```

Comment:
as discussed before, can we make it nested under `hudi-platform-service/`. Also can you clarify why we need a separate profile here?

Reply:
all right

Reply:
fix all